### ISSUE TITLE
RGBlink Q16 video processor

### ISSUE THREAD

**Issue by threethree1** [bitfocus/companion-module-requests#1515]
Device name: Q16pro video processor
Goal: To achieve the functions mentioned in the API protocol such as save, load, and call scenarios
Attached is the API protocol file
[Q16pro Gen2_Open API_V1.0_20240318.pdf](https://github.com/user-attachments/files/15864681/Q16pro.Gen2_Open.API_V1.0_20240318.pdf)

**Comment by threethree1** [comment]
Device name: RGBlink vue PTZ
Goal: To achieve the functions mentioned in the API protocol such as save, load, and call scenarios
Attached is the API protocol file
[Uploading RGBlink vue PTZ camera_APIl_EN_V1.3_20240603.docx…]()

**Comment by marcingum** [comment]
@threethree1 Have you tried using the 'Venus X3' module to connect to Q16? https://github.com/bitfocus/companion-module-requests/issues/444#issuecomment-2028807056
Some part of the API seems to be the same, for functions: power ON/OFF, save page, load page, clear page, load scene/bank.

Of course, if you try to use it, it is at your own risk!

**Comment by threethree1** [comment]
No, I haven't tried to use it, there are similar APIs, but some of them are different, so I need to add the correct model to control the device, thanks!

**Comment by threethree1** [comment]
I would also like to add the following models if possible, thanks
Device name: RGBlink X8，RGBLink D8  ，RGBLink mini-edge
Goal: To achieve the functions mentioned in the API protocol such as save, load, and call scenarios
Attached is the API protocol file
[X8 API_V1.0_20220509.pdf](https://github.com/user-attachments/files/16468974/X8.API_V1.0_20220509.pdf)
[Uploading D8-API_20240729(1)_1_40_translate_20240802142118.pdf…]()
[Uploading mini_edge API(EN)-20240221.pdf…]()

**Comment by marcingum** [comment]
Some of your links not working (D8, mini edge, vue PTZ).

Contact me by email, I'm considering writing the Q16 integration. You can find the address in my github profile.

**Comment by josephdadams** [comment]
This is now available as the **`rgblink-mini`** module (latest version v2.0.0).

You can install it from Companion's module store, or see https://bitfocus.io/connections/rgblink-mini

Closing as implemented — please open a new issue on [the module's repository](https://github.com/bitfocus/companion-module-rgblink-mini/issues) if your specific model still doesn't work.
